src/theme/CodeBlock/index.tsx

```diff
@@ -5,6 +5,12 @@ import DocsCodeBlock, { CodeBlockFallback } from "../../components/code-block";
 type Props = ComponentProps<typeof DocsCodeBlock>;
 
 export default function CodeBlock({ className, ...props }: Props) {
+	// If className is not provided, check if props.language exists. If it does,
+	// use that to construct a className, prefixed with "language-".
+	if (!className && props.language) {
```

Comment:
Similarly, when using `CodeBlock` directly, it wasn't working when just passing the language, when it should to make it easier to use. This shims that behavior so it works correctly now.